tests/helpers.rs

```diff
@@ -0,0 +1,123 @@
+use reqwest::Client;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use std::env;
+use std::panic;
+use std::process::Command;
+
+/// Helper function which does test set up and teardown
+pub async fn run_test<T>(test: T) -> ()
+where
+    T: FnOnce(TestAgentCli) -> () + panic::UnwindSafe,
+{
+    let (agent_cli, wallet_id) = setup().await;
+    let result = panic::catch_unwind(|| test(agent_cli));
+    teardown(wallet_id).await;
+    assert!(result.is_ok())
```

Comment:
It might be nice if this assert has a string explaining what went wrong

```suggestion
    assert!(result.is_ok(), "X went wrong.")
```